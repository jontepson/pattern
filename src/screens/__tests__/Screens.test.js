import {
    StartScreen,
    LoginScreen,
    CameraScreen,
    LoggedInScreen,
    MapScreen,
    DriveScreen
  } from '../'
import React from 'react';
import renderer from 'react-test-renderer';

describe('<App />', () => {
    it('<StartScreen /> renders correctly', () => {
        const tree = renderer.create(<StartScreen />).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('<CameraScreen /> renders correctly', () => {
        const tree = renderer.create(<CameraScreen />).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('<LoggedInScreen /> renders correctly', () => {
        const tree = renderer.create(<LoggedInScreen />).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('<DriveScreen /> renders correctly', () => {
        const tree = renderer.create(<DriveScreen />).toJSON();
        expect(tree).toMatchSnapshot();
    });
    
});