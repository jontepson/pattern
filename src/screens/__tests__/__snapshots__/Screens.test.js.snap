// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> <CameraScreen /> renders correctly 1`] = `
<Text>
  Requesting for camera permission
</Text>
`;

exports[`<App /> <DriveScreen /> renders correctly 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "backgroundColor": "#ffffff",
      "flex": 1,
      "width": "100%",
    }
  }
>
  <Image
    resizeMode="repeat"
    source={1}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": "100%",
        },
        undefined,
      ]
    }
  />
  <View
    onLayout={[Function]}
    style={
      Array [
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "flex": 1,
          "justifyContent": "center",
          "maxWidth": 340,
          "padding": 20,
          "width": "100%",
        },
        Object {
          "paddingBottom": 0,
        },
      ]
    }
  >
    <AIRMap
      followsUserLocation={true}
      handlePanDrag={false}
      initialRegion={null}
      onChange={[Function]}
      onLayout={[Function]}
      onMapReady={[Function]}
      onMarkerPress={[Function]}
      onUserLocationChange={[Function]}
      region={null}
      showsUserLocation={true}
      style={
        Object {
          "height": 675,
          "marginBottom": "5%",
          "marginTop": "10%",
          "width": 675,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#560CCE",
            "fontSize": 21,
            "fontWeight": "bold",
            "paddingVertical": 12,
            "textAlign": "center",
          }
        }
      >
        Current speed: 
        0
      </Text>
      <AIRMapPolyline
        coordinates={Array []}
        lineCap="round"
        lineJoin="round"
        strokeColor="#000"
        strokeWidth={1}
      />
    </AIRMap>
    <View
      style={
        Object {
          "backgroundColor": "#6200ee",
          "borderColor": "transparent",
          "borderRadius": 4,
          "borderStyle": "solid",
          "borderWidth": 0,
          "elevation": 2,
          "marginVertical": 10,
          "minWidth": 64,
          "paddingVertical": 2,
          "shadowColor": "#000000",
          "shadowOffset": Object {
            "height": 0.75,
            "width": 0,
          },
          "shadowOpacity": 0.24,
          "shadowRadius": 1.5,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityState={
          Object {
            "disabled": undefined,
          }
        }
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 4,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
              },
              undefined,
            ]
          }
        >
          <Text
            numberOfLines={1}
            selectable={false}
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontFamily": "System",
                  "fontWeight": "400",
                },
                Object {
                  "textAlign": "left",
                },
                Array [
                  Object {
                    "letterSpacing": 1,
                    "marginHorizontal": 16,
                    "marginVertical": 9,
                    "textAlign": "center",
                  },
                  undefined,
                  Object {
                    "textTransform": "uppercase",
                  },
                  Object {
                    "color": "#ffffff",
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontSize": 15,
                    "fontWeight": "bold",
                    "lineHeight": 26,
                  },
                ],
              ]
            }
          >
            Return Bike
          </Text>
        </View>
      </View>
    </View>
    <Text
      style={
        Array [
          Object {
            "color": "#000000",
            "fontFamily": "System",
            "fontWeight": "400",
          },
          Object {
            "textAlign": "left",
          },
          Object {
            "bottom": 10,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "textAlign": "center",
          },
        ]
      }
    >
      By Svenska Elsparkcyklar AB
    </Text>
  </View>
</View>
`;

exports[`<App /> <LoggedInScreen /> renders correctly 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "backgroundColor": "#ffffff",
      "flex": 1,
      "width": "100%",
    }
  }
>
  <Image
    resizeMode="repeat"
    source={1}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": "100%",
        },
        undefined,
      ]
    }
  />
  <View
    onLayout={[Function]}
    style={
      Array [
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "flex": 1,
          "justifyContent": "center",
          "maxWidth": 340,
          "padding": 20,
          "width": "100%",
        },
        Object {
          "paddingBottom": 0,
        },
      ]
    }
  >
    <RNSVGSvgView
      align="xMidYMid"
      bbHeight={100}
      bbWidth={100}
      fill="#560CCE"
      focusable={false}
      height={100}
      meetOrSlice={0}
      minX={0}
      minY={0}
      secondaryFill={null}
      secondaryOpacity={0.4}
      size={100}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
            "borderWidth": 0,
          },
          Object {},
          Object {
            "flex": 0,
            "height": 100,
            "width": 100,
          },
        ]
      }
      vbHeight={512}
      vbWidth={512}
      width={100}
    >
      <RNSVGGroup
        fill={4283829454}
        propList={
          Array [
            "fill",
          ]
        }
      >
        <RNSVGPath
          d="M152.2 236.4c-7.7-8.2-19.7-7.7-24.8 2.8L1.6 490.2c-5 10 2.4 21.7 13.4 21.7h175c5.8.1 11-3.2 13.4-8.4 37.9-77.8 15.1-196.3-51.2-267.1zM244.4 8.1c-122.3 193.4-8.5 348.6 65 495.5 2.5 5.1 7.7 8.4 13.4 8.4H497c11.2 0 18.4-11.8 13.4-21.7 0 0-234.5-470.6-240.4-482.3-5.3-10.6-18.8-10.8-25.6.1z"
          fill={4283829454}
          fillOpacity={1}
          propList={
            Array [
              "fill",
              "fillOpacity",
            ]
          }
        />
      </RNSVGGroup>
    </RNSVGSvgView>
    <Text
      style={
        Array [
          Object {
            "color": "#000000",
            "fontFamily": "System",
            "fontWeight": "400",
          },
          Object {
            "textAlign": "left",
          },
          Object {
            "color": "#560CCE",
            "fontSize": 21,
            "fontWeight": "bold",
            "paddingVertical": 12,
          },
        ]
      }
    >
      Ajjo
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#000000",
            "fontFamily": "System",
            "fontWeight": "400",
          },
          Object {
            "textAlign": "left",
          },
          Object {
            "fontSize": 15,
            "lineHeight": 21,
            "marginBottom": 12,
            "textAlign": "center",
          },
        ]
      }
    >
      No more walking
    </Text>
    <View
      style={
        Object {
          "backgroundColor": "#6200ee",
          "borderColor": "transparent",
          "borderRadius": 4,
          "borderStyle": "solid",
          "borderWidth": 0,
          "elevation": 2,
          "marginVertical": 10,
          "minWidth": 64,
          "paddingVertical": 2,
          "shadowColor": "#000000",
          "shadowOffset": Object {
            "height": 0.75,
            "width": 0,
          },
          "shadowOpacity": 0.24,
          "shadowRadius": 1.5,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityState={
          Object {
            "disabled": undefined,
          }
        }
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 4,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
              },
              undefined,
            ]
          }
        >
          <Text
            numberOfLines={1}
            selectable={false}
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontFamily": "System",
                  "fontWeight": "400",
                },
                Object {
                  "textAlign": "left",
                },
                Array [
                  Object {
                    "letterSpacing": 1,
                    "marginHorizontal": 16,
                    "marginVertical": 9,
                    "textAlign": "center",
                  },
                  undefined,
                  Object {
                    "textTransform": "uppercase",
                  },
                  Object {
                    "color": "#ffffff",
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontSize": 15,
                    "fontWeight": "bold",
                    "lineHeight": 26,
                  },
                ],
              ]
            }
          >
            Hyr cykel
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#6200ee",
          "borderColor": "transparent",
          "borderRadius": 4,
          "borderStyle": "solid",
          "borderWidth": 0,
          "elevation": 2,
          "marginVertical": 10,
          "minWidth": 64,
          "paddingVertical": 2,
          "shadowColor": "#000000",
          "shadowOffset": Object {
            "height": 0.75,
            "width": 0,
          },
          "shadowOpacity": 0.24,
          "shadowRadius": 1.5,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityState={
          Object {
            "disabled": undefined,
          }
        }
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 4,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
              },
              undefined,
            ]
          }
        >
          <Text
            numberOfLines={1}
            selectable={false}
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontFamily": "System",
                  "fontWeight": "400",
                },
                Object {
                  "textAlign": "left",
                },
                Array [
                  Object {
                    "letterSpacing": 1,
                    "marginHorizontal": 16,
                    "marginVertical": 9,
                    "textAlign": "center",
                  },
                  undefined,
                  Object {
                    "textTransform": "uppercase",
                  },
                  Object {
                    "color": "#ffffff",
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontSize": 15,
                    "fontWeight": "bold",
                    "lineHeight": 26,
                  },
                ],
              ]
            }
          >
            Karta 1
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#6200ee",
          "borderColor": "transparent",
          "borderRadius": 4,
          "borderStyle": "solid",
          "borderWidth": 0,
          "elevation": 2,
          "marginVertical": 10,
          "minWidth": 64,
          "paddingVertical": 2,
          "shadowColor": "#000000",
          "shadowOffset": Object {
            "height": 0.75,
            "width": 0,
          },
          "shadowOpacity": 0.24,
          "shadowRadius": 1.5,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityState={
          Object {
            "disabled": undefined,
          }
        }
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 4,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
              },
              undefined,
            ]
          }
        >
          <Text
            numberOfLines={1}
            selectable={false}
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontFamily": "System",
                  "fontWeight": "400",
                },
                Object {
                  "textAlign": "left",
                },
                Array [
                  Object {
                    "letterSpacing": 1,
                    "marginHorizontal": 16,
                    "marginVertical": 9,
                    "textAlign": "center",
                  },
                  undefined,
                  Object {
                    "textTransform": "uppercase",
                  },
                  Object {
                    "color": "#ffffff",
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontSize": 15,
                    "fontWeight": "bold",
                    "lineHeight": 26,
                  },
                ],
              ]
            }
          >
            Karta 2
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#6200ee",
          "borderColor": "transparent",
          "borderRadius": 4,
          "borderStyle": "solid",
          "borderWidth": 0,
          "elevation": 2,
          "marginVertical": 10,
          "minWidth": 64,
          "paddingVertical": 2,
          "shadowColor": "#000000",
          "shadowOffset": Object {
            "height": 0.75,
            "width": 0,
          },
          "shadowOpacity": 0.24,
          "shadowRadius": 1.5,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityState={
          Object {
            "disabled": undefined,
          }
        }
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 4,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
              },
              undefined,
            ]
          }
        >
          <Text
            numberOfLines={1}
            selectable={false}
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontFamily": "System",
                  "fontWeight": "400",
                },
                Object {
                  "textAlign": "left",
                },
                Array [
                  Object {
                    "letterSpacing": 1,
                    "marginHorizontal": 16,
                    "marginVertical": 9,
                    "textAlign": "center",
                  },
                  undefined,
                  Object {
                    "textTransform": "uppercase",
                  },
                  Object {
                    "color": "#ffffff",
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontSize": 15,
                    "fontWeight": "bold",
                    "lineHeight": 26,
                  },
                ],
              ]
            }
          >
            Logga ut
          </Text>
        </View>
      </View>
    </View>
    <Text
      style={
        Array [
          Object {
            "color": "#000000",
            "fontFamily": "System",
            "fontWeight": "400",
          },
          Object {
            "textAlign": "left",
          },
          Object {
            "bottom": 10,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "textAlign": "center",
          },
        ]
      }
    >
      By Svenska Elsparkcyklar AB
    </Text>
  </View>
</View>
`;

exports[`<App /> <StartScreen /> renders correctly 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "backgroundColor": "#ffffff",
      "flex": 1,
      "width": "100%",
    }
  }
>
  <Image
    resizeMode="repeat"
    source={1}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": "100%",
        },
        undefined,
      ]
    }
  />
  <View
    onLayout={[Function]}
    style={
      Array [
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "flex": 1,
          "justifyContent": "center",
          "maxWidth": 340,
          "padding": 20,
          "width": "100%",
        },
        Object {
          "paddingBottom": 0,
        },
      ]
    }
  >
    <RNSVGSvgView
      align="xMidYMid"
      bbHeight={100}
      bbWidth={100}
      fill="#560CCE"
      focusable={false}
      height={100}
      meetOrSlice={0}
      minX={0}
      minY={0}
      secondaryFill={null}
      secondaryOpacity={0.4}
      size={100}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
            "borderWidth": 0,
          },
          Object {},
          Object {
            "flex": 0,
            "height": 100,
            "width": 100,
          },
        ]
      }
      vbHeight={512}
      vbWidth={512}
      width={100}
    >
      <RNSVGGroup
        fill={4283829454}
        propList={
          Array [
            "fill",
          ]
        }
      >
        <RNSVGPath
          d="M152.2 236.4c-7.7-8.2-19.7-7.7-24.8 2.8L1.6 490.2c-5 10 2.4 21.7 13.4 21.7h175c5.8.1 11-3.2 13.4-8.4 37.9-77.8 15.1-196.3-51.2-267.1zM244.4 8.1c-122.3 193.4-8.5 348.6 65 495.5 2.5 5.1 7.7 8.4 13.4 8.4H497c11.2 0 18.4-11.8 13.4-21.7 0 0-234.5-470.6-240.4-482.3-5.3-10.6-18.8-10.8-25.6.1z"
          fill={4283829454}
          fillOpacity={1}
          propList={
            Array [
              "fill",
              "fillOpacity",
            ]
          }
        />
      </RNSVGGroup>
    </RNSVGSvgView>
    <Text
      style={
        Array [
          Object {
            "color": "#000000",
            "fontFamily": "System",
            "fontWeight": "400",
          },
          Object {
            "textAlign": "left",
          },
          Object {
            "color": "#560CCE",
            "fontSize": 21,
            "fontWeight": "bold",
            "paddingVertical": 12,
          },
        ]
      }
    >
      Ajjo
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#000000",
            "fontFamily": "System",
            "fontWeight": "400",
          },
          Object {
            "textAlign": "left",
          },
          Object {
            "fontSize": 15,
            "lineHeight": 21,
            "marginBottom": 12,
            "textAlign": "center",
          },
        ]
      }
    >
      No more walking
    </Text>
    <View
      style={
        Object {
          "backgroundColor": "#6200ee",
          "borderColor": "transparent",
          "borderRadius": 4,
          "borderStyle": "solid",
          "borderWidth": 0,
          "elevation": 2,
          "marginVertical": 10,
          "minWidth": 64,
          "paddingVertical": 2,
          "shadowColor": "#000000",
          "shadowOffset": Object {
            "height": 0.75,
            "width": 0,
          },
          "shadowOpacity": 0.24,
          "shadowRadius": 1.5,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityState={
          Object {
            "disabled": undefined,
          }
        }
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 4,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
              },
              undefined,
            ]
          }
        >
          <Text
            numberOfLines={1}
            selectable={false}
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontFamily": "System",
                  "fontWeight": "400",
                },
                Object {
                  "textAlign": "left",
                },
                Array [
                  Object {
                    "letterSpacing": 1,
                    "marginHorizontal": 16,
                    "marginVertical": 9,
                    "textAlign": "center",
                  },
                  undefined,
                  Object {
                    "textTransform": "uppercase",
                  },
                  Object {
                    "color": "#ffffff",
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontSize": 15,
                    "fontWeight": "bold",
                    "lineHeight": 26,
                  },
                ],
              ]
            }
          >
            Logga in
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#6200ee",
          "borderColor": "transparent",
          "borderRadius": 4,
          "borderStyle": "solid",
          "borderWidth": 0,
          "elevation": 2,
          "marginVertical": 10,
          "minWidth": 64,
          "paddingVertical": 2,
          "shadowColor": "#000000",
          "shadowOffset": Object {
            "height": 0.75,
            "width": 0,
          },
          "shadowOpacity": 0.24,
          "shadowRadius": 1.5,
          "width": "100%",
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityState={
          Object {
            "disabled": undefined,
          }
        }
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 4,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
              },
              undefined,
            ]
          }
        >
          <Text
            numberOfLines={1}
            selectable={false}
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontFamily": "System",
                  "fontWeight": "400",
                },
                Object {
                  "textAlign": "left",
                },
                Array [
                  Object {
                    "letterSpacing": 1,
                    "marginHorizontal": 16,
                    "marginVertical": 9,
                    "textAlign": "center",
                  },
                  undefined,
                  Object {
                    "textTransform": "uppercase",
                  },
                  Object {
                    "color": "#ffffff",
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontSize": 15,
                    "fontWeight": "bold",
                    "lineHeight": 26,
                  },
                ],
              ]
            }
          >
            Fortsätt utan att logga in
          </Text>
        </View>
      </View>
    </View>
    <Text
      style={
        Array [
          Object {
            "color": "#000000",
            "fontFamily": "System",
            "fontWeight": "400",
          },
          Object {
            "textAlign": "left",
          },
          Object {
            "bottom": 10,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "textAlign": "center",
          },
        ]
      }
    >
      By Svenska Elsparkcyklar AB
    </Text>
  </View>
</View>
`;
